Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/AdlsDeploy.java

```diff
@@ -42,7 +45,7 @@ public AdlsDeploy(@NotNull String adlsRootPath,
     }
 
     @NotNull
-    public Observable<String> deploy(@NotNull File src, @NotNull URI dest)  {
+    public Observable<String> deploy(@NotNull File src, @NotNull URI dest, @NotNull Observer<AbstractMap.SimpleImmutableEntry<MessageInfoType, String>> logSubject) {
```

Comment:
It's not good to extend the method. May output ctrl message in `SparkJob.deploy`